nullaway/src/main/java/com/uber/nullaway/handlers/Handler.java

```diff
@@ -314,6 +314,8 @@ void onDataflowVisitLambdaResultExpression(
    */
   ImmutableSet<String> onRegisterImmutableTypes();
 
+  void suggest(VisitorState state, Symbol target, ErrorMessage errorMessage);
```

Comment:
This method needs a better name and also documentation.  What is it supposed to do?

Reply:
It is suppose to create ```SuggestedFixInfo``` whenever a resolvable error is detected. Would ```serializeTypeChangeSuggestion``` work ?

Reply:
See this comment: https://github.com/uber/NullAway/pull/517#discussion_r777136253 maybe we don't even need a handler